Added Intron AfriSpeech-200 Automatic Speech Recognition Challenge

Zindi-Africa opened the pull request.

hcarlens commented:
Thanks! I fixed a typo in the competition URL and updated the prize money to $5k since that's what the competition page seems to say. Let me know if that's incorrect and I can update.

Zindi-Africa commented:
This is correct.

Thank you!

*The Zindi team*

***@***.*** ||  www.zindi.africa
Click here to become a sponsor of UmojaHack Africa 2023!
<https://umojahack.africa/?utm_source=email&utm_medium=referral&utm_campaign=umoja-hack&utm_id=UH>

<https://umojahack.africa/?utm_source=email&utm_medium=referral&utm_campaign=umoja-hack&utm_id=UH>

Building AI together


On Sun, 19 Feb 2023 at 10:55, Harald Carlens ***@***.***>
wrote:

> Thanks! I fixed a typo in the competition URL and updated the prize money
> to $5k since that's what the competition page seems to say. Let me know if
> that's incorrect and I can update.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mlcontests/mlcontests.github.io/pull/138#issuecomment-1435928972>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALHNGEX5MF7OU57B37GBZB3WYHNWTANCNFSM6AAAAAAU7HVBHY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
